src/iframe-work-arounds.js

```diff
@@ -4,6 +4,8 @@ const proto = {
             onReady: () => this.onIFrameResizerReady()
         };
 
+        window.scrollToOrig = window.scrollTo;
```

Comment:
does this still need to be here?

Reply:
Yes, otherwise it will fail outside of the iframe